test/spec/Feature/Query/DeleteSpec.hs

```diff
@@ -220,9 +220,55 @@ spec =
       it "works on a table without a pk by ordering by 'ctid'" $
         baseTable "limited_delete_items_no_pk" "id" tblDataBefore
         `mutatesWith`
-        requestMutation methodDelete "/limited_delete_items_no_pk?order=ctid&limit=1&offset=1" mempty
+        requestMutation methodDelete "/limited_delete_items_no_pk?order=ctid&limit=1&offset=1" mempty mempty
         `shouldMutateInto`
         [json|[
           { "id": 1, "name": "item-1" }
         , { "id": 3, "name": "item-3" }
         ]|]
+
+      it "ignores the Range header" $ do
+        baseTable "limited_delete_items" "id" tblDataBefore
+         `mutatesWith`
+         requestMutation methodDelete "/limited_delete_items"
+          (rangeHdrs (ByteRangeFromTo 0 0)) mempty
+         `shouldMutateInto`
+         [json|[]|]
+
+        baseTable "limited_delete_items" "id" tblDataBefore
+         `mutatesWith`
+         requestMutation methodDelete "/limited_delete_items?order=id"
+          (rangeHdrs (ByteRangeFromTo 0 0)) mempty
+         `shouldMutateInto`
+         [json|[]|]
```

Comment:
Why is this operation repeated twice?

Reply:
Ah, I added that specifically for the Limited Update/Delete features. Before this change, if the Range header is used with `order=id`, then limited update/delete works as if the ranges where limits. I think it should have it's own description instead of removing it. Because `limit` and `Range` are kinda tied together, maybe some changes could bypass this.

Reply:
Ah ok.. yeah a comment or a test with description would clarify.